### PR TITLE
samuel tiokeng

### PR DESCRIPTION
**Pull request by xaster-Kies**
Just added myself as a contributor

### CODE REVIEW

**Comment by xaster-Kies** [comment]
Hey @elhmn,  when your chanced, take a look

File: content/post/contributors/samuel-tiokeng.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title: "Samuel Tiokeng"
+date: 2022-05-06T12:25:40+01:00
+authors: ["xaster-Kies"]
+---
+
+Hello🙂👋🏽 I'm Samuel,
+
+A Software Engineering student from the University of Buea, a codenewbie. I enjoy navigating the tech space, exploring what I find intresting and sharing what I learn along the way.
```
[the remark this answers is not in the document]
Comment (xaster-Kies) [reply content/post/contributors/samuel-tiokeng.md:9]: Ok @RMPR let me do the corrections!

[the remark this answers is not in the document]
Comment (elhmn) [reply content/post/contributors/samuel-tiokeng.md:9]: You can do the corrections, by clicking on the commit suggestion button, here on the GitHub UI.
Screenshot 2022-05-07 at 08 52 49

**Comment by xaster-Kies** [comment]
Did not know about that @elhmn , thanks for mentioning. Will likely be using that now 🙂🙌🏾

**Pull request merged**